app/components/Alerts/Front-End/MissingCredentialsAlert.tsx

```
@@ -0,0 +1,34 @@
+    .filter((envVar) => !process.env[envVar])
+    .map((missingEnvVar) => ({
+      type: 'error',
+      message: `The environment variable ${missingEnvVar} is missing. Please check your configuration.`,
+      title: 'Configuration Error',
+      dismissible: true,
+    }));
+
+  return (
+    <>
+      {missingEnvVars.map((alert, index) => (
+        <Alert key={index} {...alert} />
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)